{"name": "test_someting2", "status": "failed", "statusDetails": {"message": "AssertionError: The sataus code is wrong", "trace": "self = <tests.test.crud.test_crud.TestCrud object at 0x000002D0A496C430>\nget_booking_id = 5058\n\n    def test_someting2(self,get_booking_id): ##Time pass ignore it\n        url=APIConstants.url_get_search(booking_id=get_booking_id)\n        print(url)\n        print(get_booking_id)\n        response=get_request(url=url,auth=None,in_json=False)\n        print(response.json())\n        actual_status_code=response.status_code\n>       verify_http_status_code(response_data=actual_status_code,expected_data=404)\n\ntest_crud.py:72: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nresponse_data = 200, expected_data = 404\n\n    def verify_http_status_code(response_data, expected_data):\n>       assert response_data == expected_data ,\"The sataus code is wrong\"\nE       AssertionError: The sataus code is wrong\n\n..\\..\\..\\src\\helpers\\common_verification.py:2: AssertionError"}, "attachments": [{"name": "stdout", "source": "d46e7d4a-34c1-4e92-95ef-161f10fb3262-attachment.txt", "type": "text/plain"}], "start": 1730196044914, "stop": 1730196046450, "uuid": "ab96d252-d02a-4152-acbd-ff125846e19c", "historyId": "f96621bee909390f9e3b4e6550927874", "testCaseId": "78c38a059a447b210768886c0f3862a4", "fullName": "tests.test.crud.test_crud.TestCrud#test_someting2", "labels": [{"name": "parentSuite", "value": "tests.test.crud"}, {"name": "suite", "value": "test_crud"}, {"name": "subSuite", "value": "TestCrud"}, {"name": "host", "value": "LAPTOP-J82A4UMN"}, {"name": "thread", "value": "10128-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test.crud.test_crud"}]}